graylog2-web-interface/src/views/components/aggregationbuilder/SeriesSelect.tsx

```diff
@@ -168,10 +164,8 @@ class SeriesSelect extends React.Component<Props, State> {
               options={options}
               value={series.map(_wrapOption)}
               components={_components}
-              onClose={this._onClose}
-              closeMenuOnSelect={false}
+              closeMenuOnSelect
```

Comment:
This is breaking the current behavior of the series select, which allows the user to select function and field without having to reopen the select.

Reply:
Good catch, that change slipped in.